corehq/apps/geospatial/tasks.py

```diff
@@ -9,3 +26,32 @@ def geo_cases_reassignment_update_owners(domain, case_owner_updates_dict, task_k
     finally:
         celery_task_tracker = CeleryTaskTracker(task_key)
         celery_task_tracker.mark_completed()
+
+
+@serial_task('async-index-es-docs', timeout=30 * 60, queue='background_queue', ignore_result=True)
+def index_es_docs_with_location_props(domain):
+    celery_task_tracker = get_celery_task_tracker(domain, INDEX_ES_TASK_HELPER_BASE_KEY)
+    if celery_task_tracker.is_active():
+        return
+
+    geo_case_prop = get_geo_case_property(domain)
+    query = _es_case_query(domain, geo_case_prop)
+    doc_count = query.count()
+    if doc_count > MAX_GEOSPATIAL_INDEX_DOC_LIMIT:
+        celery_task_tracker.mark_as_error()
```

Comment:
What are your thoughts on increasing the timeout of the error stored in redisfor this task ?
I think 3 days is good however a project may miss this if someone does not address this error. I feel like increasing this to say 2 weeks would give sufficient time for project to address the error by reaching out to support.

Reply:
That sounds right to me. I initially didn't want to make it too long so that it's not something that bothers the user for an extended period of time, but agreed it might be worth ensuring that the relevant users are aware of the issue.

Reply:
Addressed in [707a97d](https://github.com/dimagi/commcare-hq/pull/35126/commits/707a97dcee09d4e3dfd88e7bf5dc6cb7bc0f9bc3).